main.py

```diff
@@ -1,7 +1,53 @@
 from fastapi import FastAPI
+from pydantic import BaseModel
+from typing import List
+from redis import Redis
+from rq import Queue
+
+INVALID_OPTIONS = "The current poll only accept voting in the following options: "
 
 app = FastAPI()
 
+redis_conn = Redis()
+votes_queue = Queue('votes', connection=redis_conn)
+
+current_options = {'COBOL', 'VB', 'Delphi'}
+# TODO save dict/set into redis
+redis_conn.sadd("current_poll_options", current_options)
```

Comment:
For add values in the set structure you can use loop like a:

```
current_options = ['COBOL', 'VB', 'Delphi']
for option in current_options:
    redis_conn.sadd("current_poll_options", option)
```

So, the complexity on check_options stay on O(n) for vote options and O(1) on compare "option voted" with "available options".